### ISSUE TITLE
ReactiveList renderItem introduces an unexpected new wrapper <span /> element which can break existing UIs

### ISSUE THREAD

**Issue by adamjarling** [appbaseio/reactivesearch#1547]
**Affected Projects**
React

**Library Version:** x.y.z
3.12.4

**Describe the bug**
Looks like with the latest release, the `ReactiveList` `renderItem()` function wraps it's output in a `<span>`.  This could arguably break a lot of UIs which use CSS frameworks which follow the pattern of:

```
<div class="columns">
  <div class="column"></div>
</div> 
```

by injecting this new span element:

```
<div class="columns">
  <span id="0i8asdf-987asdf">
    <div class="column"></div>
  </span>
</div> 
```

It's broken UIs in 3 of our applications, and not sure how to work around this.  Is there any way to turn off the behavior of the new `<span>` wrap?

**To Reproduce**

View the source of any component rendered via the `renderItem()` function of the `ReactiveList` component.

https://opensource.appbase.io/playground/?path=/story/result-components-reactivelist--basic

**Expected behavior**
No extra element wraps the expected rendered component.  Or a configuration option exists to specify what type of element it is and possibility to add classes to it?

**Screenshots**
![image](https://user-images.githubusercontent.com/3020266/93136975-6520ba80-f6a2-11ea-9fbe-b0d527d5feab.png)

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser: all
 - Version: any

**Comment by bietkul** [comment]
@adamjarling This issue has been fixed in `v3.12.6`.

**Comment by adamjarling** [comment]
@bietkul Thank you so much.  ReactiveSearch is awesome and just keeps getting better.  Thanks so much for all the open-source hard work.
